packages/react-instantsearch-dom-maps/src/__tests__/withGoogleMaps.tsx

```diff
@@ -36,7 +36,7 @@ describe('withGoogleMaps', () => {
 
     const Fake = withGoogleMaps(({ google }: Props) => {
       const heatmap = new google.maps.visualization.HeatmapLayer({
-        data: [10, 20, 30],
+        data: [10, 20, 30] as any,
```

Comment:
Why do you need to cast it?

Reply:
I added an explanation here: https://github.com/algolia/react-instantsearch/blob/f065f62ec25592cf3866d99ce814297006b27696/packages/react-instantsearch-dom-maps/src/__tests__/withGoogleMaps.tsx#L39-L41

Reply:
No, we don't have access to `LatLng`.